src/views/Package/index.tsx

```diff
@@ -23,8 +23,8 @@ export function Package() {
   const [language] = useLanguage();
 
   useEffect(() => {
-    void requestAssembly(name, version, scope);
     void requestMetadata(name, version, scope);
+    void requestAssembly(name, version, scope);
```

Comment:
haha if the app was next.js we could get all the info build time 😊 but all these custom hooks are very existing

Reply:
Indeed, they do exist 😛

Reply:
Exciting***